src/radical/pilot/task_description.py

```diff
@@ -85,6 +85,12 @@ class TaskDescription(ru.TypedDict):
     :meth:`radical.pilot.TaskManager.submit_tasks` to instantiate and run
     a new task.
 
+    .. envvar:: RADICAL_TEST_ENV
```

Comment:
It doesn't make sense to have more than one directive for the same documented entity. (Also, Sphinx does not play well with it, which you'll notice if you ever accidentally include a document in more than one toctree; directives, like toctrees, establish documentation _nodes_, and nodes need to be uniquely identifiable).

If an environment variable is relevant to more than one software component, and neither component "owns" the environment variable, then they should probably just reference it as a role. If some local explanatory text is appropriate in addition to the link that the role provides, that's fine. E.g.
```
Relevant environment variables
-------------------------------
 * :envvar:`RADICAL_TEST_ENV` is used by TaskDescription to blahblah.
 * :envvar:`RADICAL_TEST_ENV2`, if defined, causes yaddayadda